framework/src/modules/interoperability/base_interoperability_endpoint.ts

```diff
@@ -154,6 +155,13 @@ export abstract class BaseInteroperabilityEndpoint extends BaseEndpoint {
 		};
 	}
 
+	// eslint-disable-next-line @typescript-eslint/require-await
+	public async getCCMSchema(_context: ModuleEndpointContext): Promise<Record<string, unknown>> {
+		return {
+			ccmSchema,
```

Comment:
maybe we can `return { schema: ccmSchema }` consistent to other endpoints 
https://github.com/LiskHQ/lisk-sdk/blob/b30d10684a8f7ebf49c07050080e251b67bd3bec/framework/src/modules/auth/endpoint.ts#L100